knowledge-base/conflicting-actions-error-in-swagger-generation-net-core.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Swagger generation conflicting actions error in .NET Core
```

Comment:
The title is not clear to me. I suggest the particular error be used as a title, for example:
"Failed to load API definition is displayed on the Swagger Generation page. "